src/api-extension/models/cart-action-factory.js

```diff
@@ -841,6 +839,12 @@ class CartActionFactory {
         result.push(builder.build());
       }
 
+      // This is a bit sus, in the new "cart item discount" test I had to add an extra
+      // action in the output to cater for this, which is an action that resets the price
+      // for an item. In my test, this action is then immediately countered by a second
+      // action that sets the price again.
+      // (See mocks/actions/update-cart-with-per-item-discount-actions.json)
+      // If I remove this another test fails so I'm keeping it for now.
```

Comment:
That's an interesting point, but I think that having this comment here is a fair thing as it should be used as inspiration for an integration.

Perhaps we only word it a bit more subjectively and less personal-toned, as an FYI to the developer that will look at this code in the future?

Reply:
Sure, no problem